assets/stylesheets/join.css

```diff
@@ -1,35 +1,245 @@
-.conn-warn {
-  animation: blink 0.6s cubic-bezier(1, 0, 0, 1) infinite alternate;
-  color:     red;
+body {
+  margin-left: 50px;
+  margin-right: 50px;
 }
 
-@keyframes blink {
-  to { opacity: 0; }
+table {
+  border-collapse: collapse;
+  border-style: hidden;
+  text-align: center;
+  margin: 20px 0 0 0;
+  width: 100%;
 }
 
-.role-select {
-  width: 80%;
+th, td {
+  border: 1px solid black;
+  border-collapse: collapse;
+  text-align: center;
+  padding: 10px 0px 10px 0px;
+}
+
+.session-status {
+  margin-bottom: 10px;
 }
 
-.session-label {
-  border-radius: 12px;
-  margin:         3px 0 3px 3px;
-  padding:        5px;
+.status-value-container {
+  font-weight: bold;
 }
 
-.session-label:not(.active) {
-  box-shadow: 0 0 8px #47daff;
+.session-row.active {
+  font-weight: bold;
+  background-color: rgba(0, 128, 0, 0.5);
 }
 
-.session-label.active {
-  background-color: #47daff;
+.session-row:hover:not(.active) {
+  background-color: rgba(0, 128, 0, 0.2);
 }
 
-.session-option {
+.row-label {
   display: none;
 }
 
+.left-column {
+  width: 50%;
+  margin-right: 30px;
+}
+
+.session-filter-box {
+  width: 100%;
+  height: 40px;
+  padding-left: 10px;
+  padding-right: 40px;
```

Comment:
Removed since I don't think the `padding-right` is doing anything. It was originally intended to position the search bar icon correctly, but I don't think it's doing anything here actually.